tests/ui/test_utils.py

```diff
@@ -83,8 +82,23 @@ def test_is_muted(mocker, msg, narrow, muted_streams, muted_topics, muted):
     model.is_muted_stream = (
         mocker.Mock(return_value=(msg.get('stream_id', '') in muted_streams))
     )
+    model.stream_dict = {
+        1: {
+            'name': 'foo',
+        },
+        2: {
+            'name': 'boo',
+        },
+        3: {
+            'name': 'zoo',
+        },
+    }
```

Comment:
Would using an existing stream fixture make sense here?

Reply:
The stream names used in its parametrize are different from `stream_dict`. I could change the stream names in a separate commit.